src/vocab.rs

```diff
@@ -75,4 +75,16 @@ impl PySequenceProtocol for PyVocab {
             Ok(words[idx as usize].clone())
         }
     }
+
+    fn __contains__(&self, word: String) -> PyResult<bool> {
+        let embeds = self.embeddings.borrow();
+        Ok(embeds
+            .vocab()
+            .idx(&word)
```

Comment:
Reminder, we should implement `word(&self) -> Option<...>` and `subword(&self) -> Option<...>` on `WordIndex`, so that these cases can be simplified to:

`.vocab().idx(&word).and_then(WordIndex::word).is_some()`

Reply:
Do you want to open the issue at `finalfusion-rust`?

Reply:
https://github.com/finalfusion/finalfusion-rust/pull/76